src/main/resources/locales/en/account-request-submitted.json

```diff
@@ -4,5 +4,5 @@
   "nextMessage": "What happens next",
   "reviewMessage": "HMCTS will review your details.",
   "moreInformationMessage": "We'll email you if we need more information or to confirm that your account has been created.",
-  "contactUsMessage": "If you do not get an email from us within 5 days, call our court and tribunals service center on 0300 303 0656."
+  "contactUsMessage": "If you do not get an email from us within 5 days, call our Courts and Tribunals Service Center on 0300 303 0656."
```

Comment:
Should this also be changed on the Welsh too? If so I reckon we might need a new translation

Reply:
changed.